src/pages/index.mdx

```diff
@@ -161,9 +161,9 @@ Interact with other Stellar developers, keep up with ecosystem standards and pro
 
 | Hubble |  |  |
 | :-- | --- | --- |
-| [Connecting to Hubble](/docs/data/hubble/analyst-guide/connecting) | Connect to Hubble using one of three methods: BigQuery UI, BigQuery SDK, or Looker Studio | Learn |
+| [Connecting to Hubble](/docs/data/hubble/developer-guide/connecting-to-bigquery) | Connect to Hubble using one of three methods: BigQuery UI, BigQuery SDK, or Looker Studio | Learn |
 | [Data catalog](/docs/data/hubble/data-catalog) | View all Hubble data catalog information. | Learn |
-| [Admin guide](/docs/data/hubble/admin-guide) | A comprehensive guide that will teach you how to run your own Hubble analytics platform | Tutorial |
+| [Developer guide](/docs/data/hubble/developer-guide) | A comprehensive guide that will teach you how to run your own Hubble analytics platform | Tutorial |
```

Comment:
The developer guide encompasses more than just running stellar-etl. It may be useful to add something about `building custom data pipelines using Hubble as a source`